components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1039,7 +1039,7 @@ public static NativeObject jsFunction_searchPaginatedAPIs(Context cx, Scriptable
 
         result = apiConsumer.searchPaginatedAPIs(newSearchQuery, tenantDomain, start, end, limitAttributes);
         if (result != null) {
-            if (newSearchQuery.startsWith(APIConstants.DOCUMENTATION_SEARCH_TYPE_PREFIX)) {
+            if (newSearchQuery.startsWith(APIConstants.DOCUMENTATION_SEARCH_TYPE_PREFIX2)) {
```

Comment:
why it's rename to document search type prefix 2?

Reply:
DOCUMENTATION_SEARCH_TYPE_PREFIX is "doc" and DOCUMENTATION_SEARCH_TYPE_PREFIX2 is "doc=". Before adding this feature, only  the doc search will fall under this category. With this feature, user is allowed to do the search based on added custom properties. Hence, there may be custom properties like "documentation", "docerina", etc. and they will also be treated as doc search at this point. To avoid that, I made the relevant changes, to check "doc=", so that only the doc search will go through this path.

Reply:
Do you see any meaningful name eather saying this prefix2?

Reply:
How about "DOC_SEARCH_TYPE_PREFIX"  or "DOCUMENTATION_SEARCH_TYPE_PREFIX_WITH_EQUALS_SIGN"  ?I prefer "DOC_SEARCH_TYPE_PREFIX".